projects/overlook.md

```diff
@@ -143,76 +166,118 @@ password: overlook2021
 
-## Accessibility
+ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
 
-* Accessibility audits should be at 100% for the dashboard.
-* A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad
-* ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
 
+## Minimum Collaboration and Professionalism Expectations
 
-## Due Date
+- Team holds daily standups throughout project.
+- Commits are atomic and frequent, effectively documenting the evolution/progression of the application. There is no more than a 10% disparity in project contributions between teammates.
+- A project board is utilized (and updated throughout the project) with Github issues and labels.
+- Team uses branches, PRs and thorough code reviews to add new code to the main branch.
+- The README is formatted well and at a minimum contains:
+- Overview of project and goals
+- Overview of technologies used, challenges, wins, and any other reflections
+- Screenshots/gifs of your app
+- List of contributors
+- Team collaborates effectively to accomplish the shared goal. Team productively and professionally works through challenges and conflicts to ensure all team members are able to be heard and contribute throughout the project.
+- Instructors are available to offer support and guidance but conversations around what is and what is not working are expected to be led by the team members themselves.
 
-Make sure you submit your project [here](https://docs.google.com/forms/d/e/1FAIpQLScsgrJD22g9WnUj7-3gXMHFSPqkk9rTt86kbRTEDGfGCIMLVA/viewform?usp=sf_link){:target='blank'} by **Tuesday of Week 6 at 12pm**.
+---
 
 # Rubric
 
-## Specification Adherence
+For the rubric sections below, you will be scored as Wow, Yes or Not Yet depending on whether you have demonstrated competency in that area. Each section lists examples of what types of things we may be looking for as demonstrations of competency. Just as there are many ways to approach code, there are many many ways to demonstate competency. The following examples are not checklists to complete! There are just some examples.
 
-* 4: The application completes all iterations above without error.
-* 3: The application completes the first 4 iterations above without error. **Note: Must be completed in order to pass.**
-* 2: The application completes the first 2 iterations and is in a usable state, but has some miscellaneous bugs.
-* 1: The application completes only the first iteration, displaying the user's data, but has no additional functionality.
+<section class="answer">
+### Functional Expectations
 
-## UI/UX & Accessibility
+- Wow: Application fulfills all requirements as well as an extension.
+- Yes: Application fulfills all requirements of iterations 1-3 without bugs. **Note: Must be completed in order to pass**
```

Comment:
Since the current functionality benchmark is that they must complete the first 4 iterations without bugs in order to pass - should we make that the bar for a Yes?

My instinct would be to keep that bar the same for now knowing we can adjust later if we need to.

Reply:
Great catch! That iteration 4 vs 3 is just an oversight on my end. 

I'll update the spec so that the WOW, as well. Since we haven't offered any other option for extensions in the past, I think it's fine to leave it for now. The Mod 2 team can adjust this in the future if they want to allow other extension options.